docs/api/utils.md

```diff
@@ -20,7 +20,7 @@ await Apify.utils.sleep(1500);
 
 -   [`utils`](#utils) : `object`
     -   [`.enqueueLinks`](#utils.enqueueLinks) ⇒ `Promise<Array<QueueOperationInfo>>`
-    -   [`.requestAsBrowser`](#utils.requestAsBrowser) ⇒ `http.IncomingMessage`
+    -   [`.requestAsBrowser`](#utils.requestAsBrowser) ⇒ `Promise<http.IncomingMessage&gt;` \| `Promise.&lt;Stream>`
```

Comment:
The type annotation looks wrong. We should probably use `Promise<http.IncomingMesage|stream.Readable>` instead of `Promise<http.IncomingMessage>|Promise<Stream>`.